pages/linux/sacct.md

```diff
@@ -18,3 +18,11 @@
 - Display elapsed time, job name, number of requested CPUs, and memory requested of a job:
 
 `sacct --jobs {{job_id}} --format={{elapsed}},{{jobname}},{{reqcpus}},{{reqmem}}`
```

Comment:
@gutjuri Should the column name be fixed here and in line 16 too?

Reply:
Are you suggesting to remove the placeholders?

Reply:
> Are you suggesting to remove the placeholders?

Yes, in case these are required columns too and not placeholders as suggested in the resolved comment above (I don't use Slurm, so IG I will wait for gutjuri's suggestion)